dist/steps/build.sh

```diff
@@ -142,24 +135,11 @@ else
   echo "Build failed, with exit code $BUILD_EXIT_CODE";
 fi
 
-#
-# Permissions
-#
-
-# Make a given user owner of all artifacts
-if [[ -n "$CHOWN_FILES_TO" ]]; then
-  chown -R "$CHOWN_FILES_TO" "$BUILD_PATH_FULL"
-  chown -R "$CHOWN_FILES_TO" "$UNITY_PROJECT_PATH"
-fi
-
-# Add read permissions for everyone to all artifacts
-chmod -R a+r "$BUILD_PATH_FULL"
-chmod -R a+r "$UNITY_PROJECT_PATH"
-
-# Add execute permissions to specific files
+# Add permissions to make app runnable
 if [[ "$BUILD_TARGET" == "StandaloneOSX" ]]; then
-  OSX_EXECUTABLE_PATH="$BUILD_PATH_FULL/StandaloneOSX.app/Contents/MacOS/*"
-  chmod +x "$OSX_EXECUTABLE_PATH"
+  ADD_PERMISSIONS_PATH=$BUILD_PATH_FULL/StandaloneOSX.app/Contents/MacOS/*
+  echo "Making the following path executable: $ADD_PERMISSIONS_PATH"
+  chmod +x $ADD_PERMISSIONS_PATH
```

Comment:
Please make sure you rebase, so that you don't end up reverting the changes from a few PRs here.